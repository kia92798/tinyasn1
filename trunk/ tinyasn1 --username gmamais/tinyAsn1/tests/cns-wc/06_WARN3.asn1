MY-MODULE DEFINITIONS AUTOMATIC TAGS::= BEGIN


	IntegerMatrix ::= SEQUENCE SIZE(2) OF SEQUENCE OF INTEGER(1..10)

	CM ::= IntegerMatrix (WITH COMPONENT (SIZE(3)))

	matrix CM ::= { {5,1,2}, {10,11,5} }	--11 is not allowed
				
	
	
END
