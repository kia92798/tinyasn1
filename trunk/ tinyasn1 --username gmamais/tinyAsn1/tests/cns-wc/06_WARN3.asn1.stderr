Warning: value 'matrix'defined in line 8 does not conform to its type constraints
